backend/server.js

```diff
@@ -1,40 +1,25 @@
-import dotenv from 'dotenv'; 
-import express from 'express';
-import connectDB from './config/database.js';
-import leaderboardRoutes from './routes/leaderboardRoutes.js';
-import rosterRoutes from './routes/rosterRoutes.js';
-import errorHandler from './middlewares/errorHandler.js';
-import cors from 'cors';
+import express from "express";
+import cors from "cors";
+import dotenv from "dotenv";
+import connectDB from "./config/database.js";
+import leaderboardRoutes from "./routes/leaderboardRoutes.js";
+import rosterRoutes from "./routes/rosterRoutes.js";
+import errorHandler from "./middlewares/errorHandler.js";
 
-dotenv.config(); 
+dotenv.config();
 
 const app = express();
 
-
-const cors = require("cors");
-
-// Allow all origins or specify the frontend's origin
 app.use(cors({
-    origin: "http://localhost:5173" // Your frontend URL
```

Comment:
whats this doing?

Reply:
there was an issue, all the data was fetched but not shown on the home page. in order to fix the issue I used cors globally in the server file.